tests/test_other.py

```diff
@@ -7993,3 +7993,19 @@ def test_single_file(self):
                     assert expect_wast == os.path.exists('a.out.wast')
                     if should_run_js:
                       self.assertContained('hello, world!', run_js('a.out.js'))
+
+  def test_emar_M(self):
+    subprocess.call(['touch', 'file1'])
+    subprocess.call(['touch', 'file2'])
+    subprocess.call(['emar', 'cr', 'file1.a', 'file1'])
```

Comment:
Better to do `Popen([PYTHON, EMAR` etc., as emar may not be in the path (may be why it fails on the bots) and python is needed for windows I think.